app/frontend/src/components/MapSearch.tsx

```diff
@@ -135,28 +154,33 @@ export default function MapSearch({
         className={classes.form}
       >
         <Autocomplete
-          label="My location"
+          label="Search for location"
           value={value}
           size="small"
           options={searchOptions.map((o) => o.name)}
           loading={searchOptionsLoading}
           open={open}
           onBlur={() => setOpen(false)}
           onChange={(e, inputValue, reason) => {
+            // setAddress(inputValue ?? "", inputValue ?? "");
             setValue(inputValue ?? "");
             searchSubmit(inputValue ?? "", reason);
           }}
-          onInputChange={(_, inputValue) => setValue(inputValue)}
+          // onInputChange={(_, inputValue) => setAddress(inputValue, inputValue)}
           freeSolo
           multiple={false}
+          // show all returned results, don't do a filter client side
+          filterOptions={(x) => x}
           disableClearable={false}
           className={classes.autocomplete}
+          getOptionDisabled={(option) => option === NO_RESULTS_TEXT}
+          helperText="Press enter to search"
         />
         <IconButton
           aria-label="Search location"
           size="small"
           onClick={() => {
-            setValue(value);
+            // setAddress(value, value);
             searchSubmit(value, "create-option");
```

Comment:
This doesn't work atm

Reply:
Oh man, thanks for noticing that! Would've been a really annoying bug

Reply:
Itsi noticed!

Reply:
Fixed